### DISCUSSION TITLE
Please improve UX when publishing

### DISCUSSION THREAD

**Discussion by normanwankowski** [mozilla/hubs#4794] [Spoke Questions] [answered]
My users have a problem creating public rooms.

When they are in Spoke they hit **Publish Scene**, they naturally think that they just published a room in the metaverse. But they haven't.
They need to open the scene and they have to create a room from it (????, we are talking about scenes and than rooms), than they need to find the settings of the room and the edit tap in order to set the room to PUBLIC. 

That is not really comfortable. People also get confused by Publish and Public. Is there a way to make it easier?

My suggestion would be to have something like this in Spoke:

Create Room   ( will publish a room to the metaverse) and under it
Public /Private  (Switch to enable public mode of the room, within Spoke during the publishing process).

So when I am ready with my work in Spoke i can directly really publish my public room.

I understand there might be other concerns like testing a scene before or seperating Spoke and Hubs, but I am just passing the feedback from my community. It would be better to have better user experience.

**Comment by matthewbcool** [comment, marked as answer]
Feel free to chime in on this issue: https://github.com/mozilla/Spoke/issues/1172

Appreciate direct testimony from your community, and suggestions very helpful feedback!
